### PR TITLE
Coretime Zombienet test

### PR DESCRIPTION
**Pull request by bkchr**
This adds a Zombienet test for Coretime.

Requires: https://github.com/paritytech/polkadot-sdk/pull/2862

### CODE REVIEW

**Review by AndWeHaveAPlan — APPROVED** [review]

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4853813

**Review by pepoviola — APPROVED** [review]

File: cumulus/pallets/parachain-system/src/lib.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ use polkadot_parachain_primitives::primitives::RelayChainBlockNumber;
 use polkadot_runtime_parachains::FeeTracker;
 use scale_info::TypeInfo;
 use sp_runtime::{
-	traits::{Block as BlockT, BlockNumberProvider, Hash},
+	traits::{Block as BlockT, BlockNumberProvider, Hash, One},
```
Comment (pepoviola) [inline cumulus/pallets/parachain-system/src/lib.rs:51]:
```suggestion
	traits::{Block as BlockT, BlockNumberProvider, Hash},
```
 Clippy https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4853813

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4854132

**Review by seadanda — APPROVED** [review]
LGTM

File: cumulus/parachains/runtimes/coretime/coretime-rococo/src/coretime.rs [hunk 2]
```diff
@@ -215,6 +222,9 @@ impl pallet_broker::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type Currency = Balances;
 	type OnRevenue = CreditToCollatorPot;
+	#[cfg(feature = "fast-runtime")]
+	type TimeslicePeriod = ConstU32<10>;
```
[the remark this answers is not in the document]
Comment (bkchr) [reply cumulus/parachains/runtimes/coretime/coretime-rococo/src/coretime.rs:225]: IDK :D Session length on Rococo local is also around 1 Minute aka 10 blocks.

**Review by eskimor — APPROVED** [review]

File: polkadot/runtime/parachains/src/coretime/mod.rs [hunk 3]
```diff
@@ -247,7 +252,7 @@ fn mk_coretime_call(call: crate::coretime::CoretimeCalls) -> Instruction<()> {
 		origin_kind: OriginKind::Superuser,
 		// Largest call is set_lease with 1526 byte:
 		// Longest call is reserve() with 31_000_000
-		require_weight_at_most: Weight::from_parts(100_000_000, 20_000),
+		require_weight_at_most: Weight::from_parts(110_000_000, 20_000),
```
Comment (eskimor) [inline polkadot/runtime/parachains/src/coretime/mod.rs:255]: Was it really that tight?

Comment (bkchr) [reply polkadot/runtime/parachains/src/coretime/mod.rs:255]: Yes. I think 106_000_000 or so was it AFAIR.

**Pull request merged**